### PR TITLE
[14.0][mig]-analytic_resource_plan

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Hi @JuanyDForgeflow is this one blocked for some reason?

**Comment by JuanyDForgeflow** [comment]
@AaronHForgeFlow no, I'll be fixing this

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Can you fix resource plan line tree view?

File: analytic_resource_plan/wizard/resource_plan_line_change_state_view.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    />
+                    <button string="Cancel" class="oe_link" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record
+        id="action_resource_plan_line_change_state_form"
+        model="ir.actions.act_window"
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan/wizard/resource_plan_line_change_state_view.xml:27]: This action is not showing when selecting the resource plan lines in a list view and clicking on More > Action Can you check?

Comment (JuanyDForgeflow) [reply analytic_resource_plan/wizard/resource_plan_line_change_state_view.xml:27]: @AaronHForgeFlow I forgot to add it to the security file, it should be working now

**Comment by JuanyDForgeflow** [comment]
> Can you fix resource plan line tree view?
> 
> 

This is fixed now

File: analytic_resource_plan/wizard/analytic_resource_plan_copy_version_view.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="analytic_resource_plan_copy_version_form" model="ir.ui.view">
+        <field name="name">analytic.resource.plan.copy.version.form</field>
+        <field name="model">analytic.resource.plan.copy.version</field>
+        <field name="arch" type="xml">
+            <form
+                string="Copy Resource Plan lines to a different version"
+                version="7.0"
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan/wizard/analytic_resource_plan_copy_version_view.xml:10]: I don't think this is necessary

File: analytic_resource_plan/wizard/resource_plan_line_change_state_view.xml (hunk 1 above)
Comment (AaronHForgeFlow) [reply analytic_resource_plan/wizard/resource_plan_line_change_state_view.xml:27]: That's good, but the actions are still not showing to me. Are they showing to you? They should show in the resource plan line list.

**Pull request closed without merging**